src/util/MetadataTypes.ts

```diff
@@ -0,0 +1,12 @@
+import { namedNode } from '@rdfjs/data-model';
+import { NamedNode } from 'rdf-js';
+
+export const RDF_TYPE: NamedNode = namedNode('http://www.w3.org/1999/02/22-rdf-syntax-ns#type');
+export const CONTENT_TYPE: NamedNode = namedNode('http://www.w3.org/ns/ma-ont#format');
+export const SLUG: NamedNode = namedNode('urn:solid:slug');
+export const LAST_CHANGED: NamedNode = namedNode('urn:solid:lastChanged');
+export const BYTE_SIZE: NamedNode = namedNode('urn:solid:byteSize');
+export const ACL_RESOURCE: NamedNode = namedNode('urn:solid:acl');
```

Comment:
Let's add some more structure here.

`ACL` should be a group of its own, so should `RDF`.

The `urn:solid` things should have a prefix of their own, such as

```
export const HTTP_SLUG: NamedNode = namedNode('urn:solid:http:slug');
```

`CONTENT_TYPE`: lets have the `MA` prefix for consistency.